<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Report</class>
 <widget class="QDialog" name="Report">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="Fback" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>720</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{
	background-color: rgb(36, 36, 36);
}</string>
   </property>
   <widget class="QWidget" name="Fback1" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>1221</width>
      <height>661</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{
	background-color: rgb(45, 45, 45);
}</string>
    </property>
    <widget class="QWidget" name="Fback2" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>1171</width>
       <height>611</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget{
	background-color: rgb(55, 55, 55);
	border-radius:20px;
}</string>
     </property>
     <widget class="QWidget" name="btnBack" native="true">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>401</width>
        <height>561</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget{
	
	background-color: rgb(65, 65, 65);
	border-radius:10px;
}</string>
      </property>
      <widget class="QPushButton" name="btnSubmit">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>170</y>
         <width>121</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Rockwell</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color:rgb(255, 213, 215);
	border-radius:25px;
	border-color: rgb(53, 69, 93);
	
	color: rgb(0, 0, 0);
}</string>
       </property>
       <property name="text">
        <string>Get Data</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_date">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>80</y>
         <width>161</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Rockwell</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
	color:rgb(255, 213, 215);
}</string>
       </property>
       <property name="text">
        <string>Year/Month</string>
       </property>
      </widget>
      <widget class="QLabel" name="txtAcademic">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>0</y>
         <width>331</width>
         <height>81</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Poor Richard</family>
         <pointsize>26</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
	color:rgb(255, 213, 215);
}</string>
       </property>
       <property name="text">
        <string>Report</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnBack_2">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>170</y>
         <width>121</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Rockwell</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color:rgb(152, 96, 100);
	border-radius:25px;
	border-color: rgb(53, 69, 93);
	
	color: gold;
}</string>
       </property>
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
      <widget class="QDateEdit" name="dateEdit">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>100</y>
         <width>321</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Cascadia Code</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QDateEdit{
	background-color:rgb(255, 131, 133);
	border-radius:20px;
	padding-left:10px;
	padding-right:10px;
	color:black;
}</string>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::ButtonSymbols::UpDownArrows</enum>
       </property>
       <property name="displayFormat">
        <string>yyyy/MM/dd</string>
       </property>
       <property name="calendarPopup">
        <bool>false</bool>
       </property>
       <property name="date">
        <date>
         <year>2024</year>
         <month>11</month>
         <day>10</day>
        </date>
       </property>
      </widget>
      <widget class="QLabel" name="totIncometxt">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>240</y>
         <width>201</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Rockwell</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
	color:rgb(255, 213, 215);
}</string>
       </property>
       <property name="text">
        <string>Total Income of the Month</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelTotalIncome">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>269</y>
         <width>321</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Cascadia Code</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{	
background-color:rgb(88, 88, 88);
	border-radius:25px;
	border-color: rgb(53, 69, 93);
	padding-left:15px;
	color:white;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="labelTotalExpense">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>370</y>
         <width>321</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Cascadia Code</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{	
background-color:rgb(88, 88, 88);
	border-radius:25px;
	border-color: rgb(53, 69, 93);
	padding-left:15px;
	color:white;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="totExpensetxt">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>340</y>
         <width>291</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Rockwell</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
	color:rgb(255, 213, 215);
}</string>
       </property>
       <property name="text">
        <string>Total Expense of the Month</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelTotalSavings">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>470</y>
         <width>321</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Cascadia Code</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{	
background-color:rgb(88, 88, 88);
	border-radius:25px;
	border-color: rgb(53, 69, 93);
	padding-left:15px;
	color:white;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="tonSavingtxt">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>440</y>
         <width>191</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Rockwell</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
	color:rgb(255, 213, 215);
}</string>
       </property>
       <property name="text">
        <string>Total Savings of the Month</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tableBack" native="true">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>20</y>
        <width>681</width>
        <height>561</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget{
	
	background-color: rgb(65, 65, 65);
	border-radius:10px;
}</string>
      </property>
      <widget class="QTableWidget" name="expenseTable">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>50</y>
         <width>251</width>
         <height>461</height>
        </rect>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
       </property>
       <column>
        <property name="text">
         <string>Category</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Amount</string>
        </property>
       </column>
      </widget>
      <widget class="QTableWidget" name="incomeTable">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>50</y>
         <width>251</width>
         <height>461</height>
        </rect>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
       </property>
       <column>
        <property name="text">
         <string>Source</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Amount</string>
        </property>
       </column>
      </widget>
      <widget class="QLabel" name="incometxt">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>20</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Rockwell</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Income</string>
       </property>
      </widget>
      <widget class="QLabel" name="expensetxt">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>20</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Rockwell</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Expense</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
